lib/setup/setupSetup.js

```diff
@@ -217,42 +220,56 @@ function Setup(options) {
                     op = 9001;
                 } else if (otype === 'redis') {
                     op = 6379;
+                    if (ohost.includes(',')) {
+                        ohost = ohost.split(',');
+                        ohost.forEach((host, idx) => {
+                            ohost[idx] = host.trim();
+                        });
+                        op = 26379;
```

Comment:
I like speaking names, especially since below it was called `oport`.

Reply:
I just used same namings as it was before and did minimal changes only ;-)